.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,55 @@
 
 For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
 For new packages please briefly describe the package or provide a link to its homepage.
+
+The following sections are entirely *optional* but filling them out at least a little bit goes a long way towards speeding up the review process, so please try to write as much as you think is reasonable.
+-->
+
+## What was tested
+
+<!--
+If you can give us confidence that you have done sufficient quality-assurance on your changes, it greatly speeds up the PR review process as it removes friction in deciding to merge.
+Please describe what steps you have taken to verify that your PR's changes work as intended.
+
+I.e. that you have executed the [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests)) and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests).
+
+Or, for functions and "core" functionality, the tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
+
+Also make sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages so that automated tools can find, execute and report them.
+
+(You don't need to write prose, bullet points are sufficient.)
+-->
+
+- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
+- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
+- [x] ... <!-- Please provide further checks you did! -->
+
+<!--
+Invite other users of the component(s) you changed to review your PR by [@-mentioning](https://github.blog/news-insights/mention-somebody-they-re-notified/) them or requesting a review if they're part of the NixOS Github org.
```

Comment:
It might be useful to tell users how to figure out who to ping for changes to existing packages. Hopefully there's a simpler way than `nix eval --expr 'let pkgs = import <nixpkgs> {}; in builtins.toString (builtins.map (maintainer: "@${maintainer.github}") pkgs.PACKAGE.meta.maintainers)' --impure --raw`.